To: vim_dev@googlegroups.com
Subject: Patch 7.3.467
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.467
Problem:    Cursor positioned wrong at the command line when regaining focus
	    and using some input method.
Solution:   Do not position the cursor in command line mode.
Files:	    src/mbyte.c


*** ../vim-7.3.466/src/mbyte.c	2011-12-08 15:09:46.000000000 +0100
--- src/mbyte.c	2012-03-07 19:36:44.000000000 +0100
***************
*** 4504,4510 ****
      vgetc_busy = TRUE;
      showmode();
      vgetc_busy = old_vgetc_busy;
!     setcursor();
      out_flush();
  }
  
--- 4504,4511 ----
      vgetc_busy = TRUE;
      showmode();
      vgetc_busy = old_vgetc_busy;
!     if ((State & NORMAL) || (State & INSERT))
! 	setcursor();
      out_flush();
  }
  
*** ../vim-7.3.466/src/version.c	2012-03-07 19:30:32.000000000 +0100
--- src/version.c	2012-03-07 19:38:22.000000000 +0100
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     467,
  /**/

-- 
hundred-and-one symptoms of being an internet addict:
154. You fondle your mouse.

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
